src/MultiFormats/Registry/Codec.cs

```diff
@@ -0,0 +1,169 @@
+            Register("protobuf", 0x50);
+            Register("rlp", 0x60);
+            Register("bencode", 0x63);
+            Register("multicodec", 0x30);
+            Register("multihash", 0x31);
+            Register("multiaddr", 0x32);
+            Register("multibase", 0x33);
+            Register("dag-pb", 0x70);
+            Register("dag-cbor", 0x71);
+            Register("git-raw", 0x78);
+            Register("eth-block", 0x90);
+            Register("eth-block-list", 0x91);
+            Register("eth-tx-trie", 0x92);
+            Register("eth-tx", 0x93);
+            Register("eth-tx-receipt-trie", 0x94);
+            Register("eth-tx-receipt", 0x95);
+            Register("eth-state-trie", 0x96);
+            Register("eth-account-snapshot", 0x97);
+            Register("eth-storage-trie", 0x98);
+            Register("bitcoin-block", 0xb0);
+            Register("bitcoin-tx", 0xb1);
+            Register("zcash-block", 0xc0);
+            Register("zcash-tx", 0xc1);
+            Register("stellar-block", 0xd0);
+            Register("stellar-tx", 0xd1);
+            Register("torrent-info", 0x7b);
+            Register("torrent-file", 0x7c);
+            Register("ed25519-pub", 0xed);
+        }
+
+        /// <summary>
+        ///   The name of the codec.
+        /// </summary>
+        /// <value>
+        ///   A unique name.
+        /// </value>
+        public string Name { get; private set; }
+
+        /// <summary>
+        ///   The IPFS code assigned to the codec.
+        /// </summary>
+        /// <value>
+        ///   Valid codes at <see href="https://github.com/multiformats/multicodec/blob/master/table.csv"/>.
+        /// </value>
+        public int Code { get; private set; }
+
+        /// <summary>
+        ///   Use <see cref="Register"/> to create a new instance of a <see cref="Codec"/>.
+        /// </summary>
+        private Codec() { }
+
+        /// <summary>
+        ///   The <see cref="Name"/> of the codec.
+        /// </summary>
+        /// <value>
+        ///   The <see cref="Name"/> of the codec.
+        /// </value>
+        public override string ToString() { return Name; }
+
+        /// <summary>
+        ///   Register a new IPFS codec.
+        /// </summary>
+        /// <param name="name">
+        ///   The name of the codec.
+        /// </param>
+        /// <param name="code">
+        ///   The IPFS code assigned to the codec.
+        /// </param>
+        /// <returns>
+        ///   A new <see cref="Codec"/>.
+        /// </returns>
+        /// <exception cref="ArgumentException">
+        ///   When the <paramref name="name"/> or <paramref name="code"/> is already defined.
+        /// </exception>
+        /// <exception cref="ArgumentNullException">
+        ///   When the <paramref name="name"/> is null or empty.
+        /// </exception>
+        public static Codec Register(string name, int code)
+        {
+            if (string.IsNullOrWhiteSpace(name))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)